packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/lambda/call-aws-service-cross-region.ts

```diff
@@ -0,0 +1,184 @@
+import { Construct } from 'constructs';
+import * as iam from '../../../aws-iam';
+import { IFunction } from '../../../aws-lambda';
+import * as sfn from '../../../aws-stepfunctions';
+import { Token, Duration } from '../../../core';
+import { CrossRegionAwsSdkSingletonFunction } from '../../../custom-resource-handlers/dist/aws-stepfunctions-tasks/cross-region-aws-sdk-provider.generated';
+
+/**
+ * Properties for calling an AWS service's API action from your
+ * state machine across regions.
+ */
+export interface CallAwsServiceCrossRegionProps extends sfn.TaskStateBaseProps {
+  /**
+   * The AWS service to call in AWS SDK for JavaScript v3 style.
+   *
+   * @see https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/
+   * @example 's3'
+   */
+  readonly service: string;
+
+  /**
+   * The API action to call.
+   *
+   * Use camelCase.
+   */
+  readonly action: string;
+
+  /**
+   * Parameters for the API action call.
+   *
+   * Use PascalCase for the parameter names.
+   *
+   * @default - no parameters
+   */
+  readonly parameters?: { [key: string]: any };
+
+  /**
+   * The resources for the IAM statement that will be added to the Lambda
+   * function role's policy to allow the state machine to make the API call.
+   *
+   * By default the action for this IAM statement will be `service:action`.
+   */
+  readonly iamResources: string[];
+
+  /**
+   * The action for the IAM statement that will be added to the Lambda
+   * function role's policy to allow the state machine to make the API call.
+   *
+   * Use in the case where the IAM action name does not match with the
+   * API service/action name, e.g. `s3:ListBuckets` requires `s3:ListAllMyBuckets`.
+   *
+   * @default - service:action
+   */
+  readonly iamAction?: string;
```

Comment:
In that case, you can use `additionalIamStatements`. I agree that this should ideally be `iamActions?: string[]`, but then we lose API compatibility with `CallAwsService` construct. I think it's okay to leave this as-is because most API only requires a single IAM action anyway . wdyt?

Reply:
Can you share an example how to use `additionalIamStatements` with another action? I also think it would be nice to add this example to the readme file.

Reply:
Sounds good, thank you!